src/Courses/CourseInterfaces.tsx

```diff
@@ -107,12 +108,32 @@ export class TopicAssessmentFields {
     // showTotalGradeImmediately: boolean = false;
     // hideProblemsAfterFinish: boolean = false;
     // randomizeOrder: boolean = false;
+    studentTopicAssessmentOverride?: StudentTopicAssessmentOverrideFields[];
 
     public constructor(init?:Partial<TopicAssessmentFields>) {
         Object.assign(this, init);
     }
 }
 
+export class StudentTopicAssessmentOverrideFields {
+    // TODO fixed truncated fields from backend
+    topicAssessm?: number;
```

Comment:
😰

Reply:
Haha yeah I was telling drew about this
We have seen issues like this before where sequelize truncates fields
Since we are joining several levels deep it seems we reached the alias limit they have

I have a few ideas to rectify this but not now
1. See if sequelize has a configuration for this
2. See if using the separate queries flag in the include solves the issue
3. Use a mixin to fetch it instead